### ISSUE TITLE
[Winter Vacation]  Attention mask warning in LLM should be fixed

### ISSUE THREAD

**Issue by MapleAllen** [RockYYY888/ai-digital-signage#16]
During execution of LLM.py, a warning message is generated:
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.

Find where the problem and fix it.
